src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20210713140558_EES2401RemoveThemeAndTopicTables.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class EES2401RemoveThemeAndTopicTables : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_Publication_Topic_TopicId",
+                table: "Publication");
+
+            migrationBuilder.DropTable(
```

Comment:
I haven't reviewed this yet but I can see it doesn't contain the infrastructure update for altering data factory `pl_release_statistics_stage1` which relies on copying theme and topics.

It also doesn't appear to drop the `UpsertTheme` and `UpsertTopic` stored procedures.

Also just curious if we could drop Publication?

Reply:
Publication is now also removed, along with removing stored procedures UpsertTheme/Topic/Publication.